R/load_forecasts_zoltar.R

```diff
@@ -89,7 +87,7 @@ load_forecasts_zoltar <- function (
       )) %>%
       dplyr::select(model, forecast_date, location, horizon, temporal_resolution,
                     target_variable, target_end_date, type, quantile, value) %>%
-      dplyr::left_join(covidHubUtils::hub_locations, by=c("location" = "fips"))
+      join_with_hub_locations(hub = hub)
```

Comment:
should we also add a `hub` column in the resulting data frame?

Reply:
I would assume this is obvious as people specify which hub to load from and also see the locations